README.md

```diff
@@ -296,6 +296,7 @@ All configuration fields are required.
 *staging_branch* | The name of the bot-maintained git branch used for testing PR changes as if they were merged into their target branch. | auto
 *necessary_approvals* | The minimal number of core developers required for a PR to be merged. PRs with fewer votes are not merged, regardless of their age. | 1
 *sufficient_approvals* | The minimal number of core developers required for a PR to be merged fast (i.e., without waiting for `config::voting_delay_max`) | 2
+*core_developers* | The comma-separated list of login=id pairs representing GitHub core developers. | ""
```

Comment:
Please do not forget to also adjust the `From the bot point of view, a core developer is...` paragraph in the beginning of README.md. I suggest: `From the bot point of view, a core developer is a GitHub user listed in the `config::core_developers` configuration parameter.`